packages/dds/tree/src/simple-tree/core/treeNodeSchema.ts

```diff
@@ -188,6 +226,11 @@ export interface TreeNodeSchemaCore<
 	 * @system
 	 */
 	readonly childTypes: ReadonlySet<TreeNodeSchema>;
+
+	/**
+	 * Gets the user-provided {@link NodeSchemaMetadata} for this node.
```

Comment:
I think properties should be documented with noun phrases (and functions with verb phrases). Also this is not a node. You could either say "for this schema" for "for nodes of this type". Since its possible for some of the metadata to be specifically about the schema, I went with that:

```suggestion
	 * User-provided {@link NodeSchemaMetadata} for this schema.
```